web/modules/weather_data/src/Service/WeatherAlertTrait.php

```diff
@@ -104,7 +104,13 @@ public function getAlertsForLatLon($lat, $lon)
             $output->instruction = self::fixupNewlines(
                 $output->instruction ?? false,
             );
+
             $output->areaDesc = self::fixupNewlines($output->areaDesc ?? false);
+            if ($output->areaDesc) {
+                $output->areaDesc = array_map(function ($description) {
+                    return trim($description);
+                }, explode(";", $output->areaDesc));
```

Comment:
Ugh, the way PHP does this is just so ugly. Is it so wrong to want something more fluent?